### ISSUE TITLE
speed limit overlays are not clear over trees

### ISSUE THREAD

**Issue by kianzarrin** [CitiesSkylinesMods/TMPE#1227]
![image](https://user-images.githubusercontent.com/26344691/145626080-9a48404f-2636-4525-ba04-5b861a8fa8c4.png)
as you can see in the image above the speed limit overlay is not very clear.
Is there any need to fix this issue?

**Comment by kianzarrin** [comment]
this is with 0 transparency:
![image](https://user-images.githubusercontent.com/26344691/145626377-a45947ca-3755-4a13-9e20-8738735d0abb.png)

**Comment by aubergine10** [comment]
Is this specifically due to the mod settings UI allowing transparency value to be set too low (eg. should there be a min transparency enforced)?

**Comment by kianzarrin** [comment]
> Is this specifically due to the mod settings UI allowing transparency value to be set too low (eg. should there be a min transparency enforced)?

@aubergine10  I edited the picture. latest version and max transparency this time.

**Comment by kianzarrin** [comment]
One solution to this would be to show the speed limit on both side although is confusing. maybe change color?

**Comment by aubergine10** [comment]
When editing by segment (rather than lane) what if speed sign was a bit bigger?

I noticed that when editing by lane the signs get bigger than when editing by segment. That could be flipped? Related: https://github.com/CitiesSkylinesMods/TMPE/issues/1221#issuecomment-990229889

**Comment by kianzarrin** [comment]
@kvakvs is this still an issue?

**Comment by kvakvs** [comment]
It probably is the same, nothing changed in the game and in the rendering code.
Since Sunset Harb the overlay render order has changed it is now after UI but somewhere possibly before the street names overlay.
Not sure if we have any control over render order.

**Comment by krzychu124** [comment]
It's impossible for GUI calls to be before anything. They were/are always **after** everything so on top of already rendered things (UI/3d scene), cannot be reordered.

**Comment by krzychu124** [comment]
yeah, but as you've said that requires a shader so completely different use case. With custom shaders you can do everything.
